Sources/Internal/WebSocketConnection.swift

```diff
@@ -96,7 +96,7 @@ class WebSocketConnection {
 
     deinit {
         session.invalidateAndCancel()
-        pingTimer?.invalidate()
+        self.pingTimer?.invalidate()
```

Comment:
I think that self is not needed because we're in the deinit and have access to the properties

Reply:
I agree